invenio_audit_logs/services/service.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Audit Logs Service API."""
+
+from flask import current_app, request
+from invenio_accounts.proxies import current_datastore
+from invenio_records_resources.services.records import RecordService
+from invenio_records_resources.services.uow import unit_of_work
+from invenio_search.engine import dsl
+
+from .uow import AuditLogOp
+
+
+class AuditLogService(RecordService):
+    """Audit log service layer."""
+
+    def _get_user_context(self, identity):
+        """Return user context."""
+        return {
+            "user_account": current_datastore.get_user(identity.id),
+            "ip_address": request.headers.get("REMOTE_ADDR", request.remote_addr),
+            "session": request.cookies.get("SESSION", request.cookies["session"]),
```

Comment:
@slint @ntarocco  this kinda shouldn't happen in the service since I remember we said service is not request aware, WDYT?